include/freerdp/settings.h

```diff
@@ -919,10 +918,9 @@ struct rdp_settings
 	ALIGN64 char* Domain;                  /* 23 */
 	ALIGN64 char* PasswordHash;            /* 24 */
 	ALIGN64 BOOL WaitForOutputBufferFlush; /* 25 */
-	ALIGN64 UINT32 MaxTimeInCheckLoop;     /* 26 */
-	ALIGN64 char* AcceptedCert;            /* 27 */
-	ALIGN64 UINT32 AcceptedCertLength;     /* 28 */
-	UINT64 padding0064[64 - 29];           /* 29 */
+	ALIGN64 char* AcceptedCert;            /* 26 */
```

Comment:
Please do never move stuff in the stable API zone! Replace with a `padding` variable

Reply:
ok.